docs/book/application-integration/stand-alone.md

```diff
@@ -0,0 +1,300 @@
+# Stand-Alone
+
+The paginator can also be used stand-alone, outside of a Mezzio or laminas-mvc
+application.
+
+The example uses the following directory structure:
+
+```treeview
+example-app/
+|-- public/
+|   `-- index.php
+|-- templates/
+|   `-- pagination-control.phtml
+`-- vendor
+    `-- …
+```
+
+## Create and Configure Paginator
+
+[Create a paginator and a related adapter](../usage.md#paginating-data-collections),
+set the item count for one page and the current page number in `public/index.php`:
+
+```php
+// Create paginator
+$paginator = new Laminas\Paginator\Paginator(
+    new Laminas\Paginator\Adapter\ArrayAdapter($albums)
+);
+
+// Configure paginator
+$paginator->setItemCountPerPage(4);
+$paginator->setCurrentPageNumber((int) $_GET['page'] ?: 1);
```

Comment:
this is small detail, but when `page` is not set in query we will get notice here.

it might be better to use `??` operator instead.

Reply:
Definitely correct here!

Reply:
so we need also additional parenthesis, sorry for that:

https://3v4l.org/KLduP